src/components/pages/Login.jsx

```diff
@@ -41,41 +44,41 @@ export const Login = (props) => {
         Great to have you here with us!<br />
         Drop by every time you want.
       </Block>
-      <FirebaseContext.Consumer>
-      {firebase => <List form>
-        <ListInput
-          floatingLabel
-          label="E-mail"
-          type="email"
-          placeholder="Your e-mail"
-          value={userName}
-          onInput={(e) => {
-            setUserName(e.target.value);
-          }}
-        />
-        <ListInput
-          floatingLabel
-          label="Password"
-          type="password"
-          placeholder="Your password"
-          value={password}
-          onInput={(e) => {
-            setPassword(e.target.value);
-          }}
-        />
-      <Block>
-          <Button large fill onClick={() => signIn(firebase)}>Log in</Button>
-          <Divider text="or" color="lightGray" className="padding-top padding-bottom" />
-          <GoogleLoginButton onClick={signInSocial} style={socialMediaButtonStyles} />
-          <FacebookLoginButton onClick={signInSocial} style={socialMediaButtonStyles} />
-      </Block>
-      </List>}
-    </FirebaseContext.Consumer>
+      <FirebaseConsumer>
+        {props => <List form>
```

Comment:
Nie powinieneś zmieniać tej nazwy na props, gdyż przykrywasz nazwę props komponentu. Przez co jest to mało czytelne i można mieć wątpliwości czy poniżej chcesz przesłać propsy z firebase'a czy może propsy komponentu. Nadaj jakąś inną nazwę.